pkg/sectigo/endpoints_test.go

```diff
@@ -39,7 +39,7 @@ func TestModifyBaseURL(t *testing.T) {
 	require.Equal(t, "https://iot.sectigo.com/api/v1/certificates/find", ep.String())
 
 	// Create a mock Server
-	require.NoError(t, mock.Start())
+	require.NoError(t, mock.Start(""))
```

Comment:
I may find this out later on - but is there a default if you pass empty string or does it just work with an empty profile? If there is a default we may just want to add a comment here.